packages/cli-app/src/server/auth.js

```diff
@@ -3,17 +3,23 @@
 //
 
 import debug from 'debug';
+import fs from 'fs';
+import yaml from 'js-yaml';
 
 import { BASE_URL } from '../config';
 import { generateQRCode, verifyToken } from '../util/OTP';
+import { createPath } from './server';
 
 const log = debug('dxos:cli-app:server:auth');
 debug.enable('dxos:*');
 
 const COOKIE_MAX_AGE = 60;
 
+const whitelistFile = '~/.wire/keyhole-whitelist.yml';
```

Comment:
Looks for a constant with `~/.wire`, it's gotta be somewhere.

Reply:
Doesn't look like it, almost all file paths are specified as `~/.wire/<FIlENAME>`

Reply:
That's bad. Can you create an issue to change it?